specificatie/zoek-personen.yaml

```diff
@@ -192,7 +192,7 @@ components:
             huisnummertoevoeging:
               $ref: 'verblijfplaats.yaml#/components/schemas/Huisnummertoevoeging'
             straat:
-              $ref: 'filter.yaml#/components/schemas/StraatFilter'
```

Comment:
Wat @MelvLee al aangaf: graag "StraatFilter" blijven noemen, omdat zowel in straatnaam als in openbare ruimtenaam gezocht wordt.

Reply:
Aangepast

Reply:
Aangepast